src/parachain/replace.ts

```diff
@@ -189,9 +182,9 @@ export class DefaultReplaceAPI extends DefaultTransactionAPI implements ReplaceA
         await this.sendLogged(requestTx, this.api.events.replace.AcceptReplace);
     }
 
-    async execute(requestId: string, btcTxId?: string, merkleProof?: Bytes, rawTx?: Bytes): Promise<void> {
-        const parsedRequestId = this.api.createType("H256", "0x" + requestId);
-        [merkleProof, rawTx] = await getTxProof(this.electrsAPI, btcTxId, merkleProof, rawTx);
+    async execute(requestId: string, btcTxId?: string, merkleProof?: Bytes, rawTx?: Bytes, electrsAPI?: ElectrsAPI,): Promise<void> {
```

Comment:
Similar issue as `issue.execute()`. Possibly a workaround could be to have the correct electrs URL defined as part of the bitcoin network and define `getTxProof` as follows:

`getTxProof(btcTxId, electrsAPI?)`
-> if only the btxTxId is given, it'll fallback to the default URL of the connected BTC testnet. If electrsAPI is given it uses that.
-> The `merkleProof` and `rawTx` parameter seem necessary, why do we need those?